src/lib.cairo

```diff
@@ -1,3 +1,10 @@
 mod channel;
 mod common;
+mod input_structs;
 mod structs;
+
+use cairo_verifier::input_structs::stark_proof::StarkProof;
+
+fn main(stark_proof: StarkProof) -> (felt252, felt252) {
```

Comment:
I think for now it's ok to leave the main function in the main `lib` file as it is quite simple, but if it starts getting more complex we should move it to a separate file